change the default value of `tidb_enable_historical_stats` to `OFF` (#17892)
pingcap/docs: system-variables.md

- Persists to cluster: Yes
- Applies to hint [SET_VAR](/optimizer-hints.md#set_varvar_namevar_value): No
- Type: Boolean

> - Default value: `OFF`. Before v8.2, the default value is `ON

Reviewer: ```suggestion
- Default value: `OFF`. Before v8.2.0, the default value is `ON`.
```

> - This variable controls whether to enable historical statistics. The default value changes from `OFF` to `ON`, which means that historical statistics are enabled by default.

Reviewer 2: ```suggestion
- This variable controls whether to enable historical statistics. The default value is `OFF`, which means that historical statistics are disabled by default.
```
